modules/support-gather-data.adoc

```diff
@@ -5,39 +5,33 @@
 [id="support_gathering_data_{context}"]
 = Gathering data about your cluster for Red Hat Support
 
-When opening a support case, it is often helpful to provide debugging
```

Comment:
I don't find this page in the build preview.

Reply:
It is here: http://file.bos.redhat.com/pousley/091019/cnv-2.1-must-gather/cnv/cnv_users_guide/cnv-collecting-cnv-data.html#about-must-gather_cnv-collecting-cnv-data

Reply:
The list of CNV diagnostic information is detailed in this section:

http://file.bos.redhat.com/pousley/091019/cnv-2.1-must-gather/cnv/cnv_users_guide/cnv-collecting-cnv-data.html#cnv-about-collecting-cnv-data_cnv-collecting-cnv-data

Reply:
@masayag Let me explain exactly what I mean:
This file (modules/support-gather-data.adoc) contains the text string _"You can gather debugging information about your cluster"_.
When I open the preview build, using the link you provided in your comment, I search for this string, but it is not found there. Is it possible that the preview build is not updated, and therefore not aligned with what exists in this PR?

Reply:
Hi @yossisegev and @masayag - thank you both for reviewing my PR! 

To clarify, the file `support-gather-data.adoc` is part of the generic (non-CNV) OpenShift documentation about must-gather, and it can be found here: http://file.bos.redhat.com/pousley/091019/cnv-2.1-must-gather/support/gathering-cluster-data.html 

I added the About section to that page and made one or two other minor changes which is why it shows up in this PR.

The build preview is up to date. I apologize for the confusion.

Reply:
@ousleyp @masayag 
OK, I reviewed the build preview now, and it is aligned with the PR + the steps specified there are valid.
Thanks for the cooperation!